File: back/DevC/src/main/java/com/back/devc/domain/member/member/util/MemberDisplayUtil.java
package com.back.devc.domain.member.member.util;

import com.back.devc.domain.member.member.entity.Member;
import com.back.devc.domain.member.member.entity.MemberStatus;

public final class MemberDisplayUtil {

    private MemberDisplayUtil() {
    }

    public static String getDisplayName(Member member) {
        if (member == null) {
            return null;
        }

        if (member.getStatus() == MemberStatus.WITHDRAWN) {
            return "탈퇴한 회원";
        }

        return member.getNickname();
    }
}

File: back/DevC/src/main/kotlin/com/back/devc/domain/member/member/util/MemberDisplayUtil.kt
package com.back.devc.domain.member.member.util

import com.back.devc.domain.member.member.entity.Member
import com.back.devc.domain.member.member.entity.MemberStatus

object MemberDisplayUtil {

    @JvmStatic
    fun getDisplayName(member: Member?): String {
        if (member == null) {
            return ""
        }

        if (member.status == MemberStatus.WITHDRAWN) {
            return "탈퇴한 회원"
        }

        return member.nickname
    }
}
